OOP-Assignment1-Charles/OOP-Assignment1-Charles/Analyse.cs

```diff
@@ -6,31 +6,193 @@
+
+
+
+            foreach (var item in toAnalyse)
+            {
+
+                if (item.ToString() == fullStop)
+                {
+                    totalSentances++;// When a full stop is found. The sentacne count increments by one.
+                }
+
+            }
+
+
             //2. Number of vowels
+
+
+
+
+            for (int i = 0; i < vowels.Length; i++)
+            {
+                for (int j = 0; j < toAnalyse.Length; j++)
+                {
+                    if (vowels[i] == toAnalyse[j])
+                    {
+                        totalVowels++;
+                    }
+                }
+            }
+
+
+
+
             //3. Number of consonants
+
+            for (int i = 0; i < consonants.Length; i++)
+            {
+                for (int j = 0; j < toAnalyse.Length; j++)
+                {
+                    if (consonants[i] == toAnalyse[j])
+                    {
+                        totalconsonants++;
+                    }
+                }
+            }
+
+
             //4. Number of upper case letters
+
+            for (int i = 0; i < alphabet.Length; i++)
+            {
+                for (int j = 0; j < toAnalyse.Length; j++)
+                {
+                    //Console.WriteLine($"{alphabet[i].ToUpper()} {toAnalyse[j].ToString()}");
+                    if (alphabet[i].ToUpper() == toAnalyse[j].ToString())
+                    {
+
+                        totalUpper++;
+                    }
+                }
+            }
+
             //5. Number of lower case letters
-            List<int> values = new List<int>();
-            //Initialise all the values in the list to '0'
-            for (int i = 0; i < 5; i++)
+
+
+            for (int i = 0; i < alphabet.Length; i++)
             {
-                values.Add(0);
+                for (int j = 0; j < toAnalyse.Length; j++)
+                {
+                    //Console.WriteLine($"{alphabet[i].ToUpper()} {toAnalyse[j].ToString()}");
+                    if (alphabet[i].ToLower() == toAnalyse[j].ToString())
+                    {
+
+                        totalLower++;
+                    }
+                }
             }
 
+            //Option 2 long words
+            string[] indivualWords = { };
+            indivualWords = toAnalyse.Split(' ');
+            List<string> largeWords = new List<string>();
 
+            if (toAnalyse == "file")
+            {
+                foreach (string item in indivualWords)
+                {
+                    if (item.Length >= 7)
+                    {
+                        largeWords.Add(item);
+                    }
+                }
+                try
+                {
+                    System.IO.File.WriteAllLines("Longwords.txt", largeWords.ToArray());
+                }
+                catch
+                {
+                    Console.WriteLine("Sorry, we couldnt write your long words to a file ");
+                }
+
```

Comment:
Try catch here will never catch an error, as the file will always be written, no matter what is in there.